pkg/restapi/issuer/operation/operations.go

```diff
@@ -390,28 +396,69 @@ func (c *Operation) prepareCreateCredentialRequest(subject map[string]interface{
 	delete(subject, "updated_at")
 	delete(subject, "userid")
 
-	req := &createCredential{
-		Context: []string{credentialContext},
-		Subject: subject,
-		Type:    []string{"VerifiableCredential", info.Scope},
-		Profile: vcsProfile,
+	profileResponse, err := c.retrieveProfile(vcsProfile)
```

Comment:
Does edge-sandbox still create profiles as part of set-up? Is so it is weird to retrieve profile every single time we want to create a credential. We should cache profiles.

Reply:
Yes, no changes there. And agree with the caching part in non-sandbox environments.

Reply:
We should do it in sandbox too, profile is not something that changes. Sandbox issuer owns profile changes anyway. You added retrieving profile in this PR, this was not there before.

Reply:
You can add it as separate task and someone can pick it up (if we get there). We should not be calling profile there.